### PR TITLE
frontend of login, register, home, post

### PR DESCRIPTION
**Pull request by adrean-hipe**
### OVERVIEW

- added splash page blade, and method in controller
- added css for designs
- modified login blade according to design at https://www.figma.com/proto/OVaZt6dgBay07bNisWaye2/Microblog-Project-Design?type=design&node-id=193-15&t=cODMeA2s3Nqxkl2V-0&scaling=min-zoom&page-id=153%3A12
- modified register blade
- modified comment blade
- modified post blade
- modified home blade
- added partial for follow in blades
- added links for fontawesome and google fonts
- modified show blades
- modified post action blade

### NOTES

project still works as usual after modifying the frontend

### SNIPPETS

splash page
image
login page
image
register page
image
home page
image
show post
image

### CODE REVIEW

File: laravel-app/public/assets/css/app.css [hunk 1]
```diff
@@ -1,13 +1,236 @@
...
+	max-width: 100%; 
+	height: auto;
+}
+.text-share {
+	font-size: var(--font-size-smi);
+	color: var(--color-dimgray);
+}
+.text-share-link {
+	text-decoration: underline !important;
+}
```
Comment (bpocrafael) [inline laravel-app/public/assets/css/app.css:236]: add new line

**Comment by bpocrafael** [comment]
why do you need a splash page?

**Comment by adrean-hipe** [comment]
The splash page is not really needed sir but it's nice addition for distinguishing the root directory and the login page

**Comment by bpocrafael** [comment]
hmm, is it splash page tho? maybe there is another term for that.

**Comment by adrean-hipe** [comment]
> hmm, is it splash page tho? maybe there is another term for that.

Done sir. Renamed to welcome page as it fulfills the function for welcoming users for the page instead of splash page

File: laravel-app/public/assets/images/coffee-vector-min.png
Comment (hipe01mark) [inline laravel-app/public/assets/images/coffee-vector-min.png:1]: Why not converted to webp file? Check the devs/tips sharing group.

File: laravel-app/resources/views/welcome.blade.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	<div class="container">
+		<div class="row justify-content-center">
+			<div class="col-md-6">
+				<div class="text-center">
+				<img class="mr-0" src="{{ asset('assets/images/coffee-vector-min.png') }}" alt="Coffee vector">
+					<div class="logo">
+                        <span class="mi">Mi</span>
+                        <span class="cro">cro</span>
+                        <span class="blog">blog</span>
+                    </div>
```
Comment (hipe01mark) [inline laravel-app/resources/views/welcome.blade.php:15]: fix indentions

Comment (adrean-hipe) [reply laravel-app/resources/views/welcome.blade.php:15]: the indentions are okay sir in the file but displays incorrectly here only
image

File: laravel-app/public/assets/images/coffee-vector-min.png
Comment (adrean-hipe) [reply laravel-app/public/assets/images/coffee-vector-min.png:1]: Done sir. Converted to webp for better file size compression

**Review by rafael-parayno — APPROVED** [review]

**Review by bpocrafael — APPROVED** [review]

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

**Pull request merged**